src/roaring64.c

```diff
@@ -1395,6 +1395,13 @@ bool roaring64_bitmap_iterate(const roaring64_bitmap_t *r,
     return true;
 }
 
+void roaring64_bitmap_to_uint64_array(const roaring64_bitmap_t *r,
+                                      uint64_t *out) {
+    roaring64_iterator_t it = {0};
+    roaring64_iterator_init_at(r, &it, out);
```

Comment:
`roaring64_iterator_init_at`'s last parameter is `bool first`, I think we just want to always pass true here, not `out` (though presumably, `out` will always be non-null, though it may be valid to call this with null if the bitmap is empty)

Reply:
Facepalm. Thanks!